### PR TITLE
LDA with hellinger distance similarity

### PR DESCRIPTION
**Pull request by ayush-1506**
Fixes: #712

### CODE REVIEW

**Comment by ayush-1506** [comment]
Update: Changed similarity metric to Hellinger distance (it's supposed to work better in case of probability distributions.)

**Comment by ayush-1506** [comment]
Results:
```
Recall @ 1: 15.384615384615385%
Recall @ 5: 17.52136752136752%
Recall @ 10: 17.735042735042736%
Precision @ 1: 0.7352941176470588%
Precision @ 5: 2.941176470588233%
Precision @ 10: 2.034313725490193%
Recall: 17.735042735042736%
Precision: 2.034313725490196%
MAP@k : 6.410626750700279%
```

File: bugbug/similarity.py [hunk 1]
```diff
@@ -526,6 +528,51 @@ def get_distance(self, query1, query2):
...
+
+        distances.sort(key=lambda v: v[1])
+
+        return [distance[0] for distance in distances[:10]]
+
+    def get_distance(self, query1, query2):
+        raise NotImplementedError
+
+
 model_name_to_class = {
```
Comment (marco-c) [inline bugbug/similarity.py]: You'll need to add it to `model_name_to_class`, right? (Same for BM25, but you'll need to do that in another PR).

Comment (ayush-1506) [reply bugbug/similarity.py]: Yes, I'll add them in a new PR once this is merged.
